bindings/nodejs/tests/wallet/wallet.spec.ts

```diff
@@ -118,11 +118,6 @@ describe('Wallet', () => {
 
         expect(account.getMetadata().index).toStrictEqual(0);
 
-        const client = await wallet.getClient();
-
-        const localPoW = await client.getLocalPow();
```

Comment:
Please keep this getClient() so it still gets tested, just change to get bech32 hrp or something similar